gensrc/thrift/AgentService.thrift

```diff
@@ -104,6 +104,13 @@ struct TAlterTabletReqV2 {
     // version of data which this alter task should transform
     5: optional Types.TVersion alter_version
     6: optional Types.TVersionHash alter_version_hash // Deprecated
+    7: optional list<TAlterMaterializedViewParam> materialized_view_params
+}
+
+struct TAlterMaterializedViewParam {
+    1: required string column_name
+    2: required string origin_column_name
+    3: optional string mv_expr
```

Comment:
Why not using TExpr?

Reply:
The materialized view is a calculation of an expression.But we construct materialized view in olap/schema_change.cpp. we can't use expr and exec in olap layer now.So I used some hard coding to deal with this problem.May be we can refactor this part of the code until the uniform expression evaluates the logic.